lib/copy/__tests__/copy-preserve-time.test.js

```diff
@@ -47,8 +47,34 @@ describeIf64('copy', () => {
           done()
         })
       })
+
+      describe('> with a read-only file', () => {
+        const src = path.join(SRC_FIXTURES_DIR, readOnlyFile)
+        const dest = path.join(TEST_DIR, readOnlyFile)
+
+        beforeEach(() => assertPermission(src, '555', {
+          tryToSet: true,
```

Comment:
Would you please take a look at this test I've just added. 

I'm not sure what's going wrong. It works locally on my Windows Linux Bash, which is all I have access to currently. 

I'm setting permissions on a new file `b-file` to read only (444 wasn't working for me (windows bash) so I chose 555). 

I wasn't sure whether git allowed committing files with permissions so I wrote this function to actually set the file permissions in (before) the test itself. 

Travis seems to be complaining that it's setting permission to `644` instead of `555`, and this is the bit I'm not sure why.